service/web/server/src/lib/messaging.js

```diff
@@ -39,7 +48,11 @@ class Messaging {
       this.subscribe(responseChannel)
       this.sendWorkerMessage(requestChannel, requestID, method, params)
       responseTimeout = setTimeout(() => {
-        reject(new Error(`Timeout while sending command '${method}' to testid '${workerID}'`))
+        reject(
+          new Error(
+            `Timeout for request: '${requestID}', with method:'${method}', sent to testid: '${workerID}'`,
```

Comment:
requestID can be (should be) matched to a message received by a worker running a test. Using this log (which will show up in server logs) we can track down, was a message recieved by a worker, and was a response sent. If not, we get this log and can figure out where in the chain it was dropped.